.github/workflows/python-publish.yml

```diff
@@ -10,8 +10,8 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-python@v2
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
```

Comment:
Dependabot and renovate can also update these, might be an option if you don't want to update them manually.

Reply:
That's a good idea. Probably it's worth it to consider Dependabot for all sorts of other dependencies on this project. I'd still merge this PR for now.